src/NuGet.Clients/NuGet.Tools/NuGetTools.vsct

```diff
@@ -258,8 +264,10 @@
       <IDSymbol name="IDG_VENUS_CTX_REFERENCE" value="27" />
       <IDSymbol name="IDG_CTX_REFERENCE" value="0x102" />
     </GuidSymbol>
+    <!--guidReferenceContext is aka: ShellMainMenu_guid -->
     <GuidSymbol name="guidReferenceContext" value="{D309F791-903F-11D0-9EFC-00A0C911004F}">
-      <IdSymbol name="cmdAddReferenceGroup" value="0x450" />
+      <IDSymbol name="cmdAddReferenceGroup" value="0x450" />
```

Comment:
Wh some IDSymbol have names in camelCase and other with capital letters and underscore?

Reply:
Can you point to an example of one that isn't "IDSymbol" ?

Reply:
I see now that you meant the name property. Some names are globally defined in VS, and they follow the all-caps and underscore pattern. If you think there's an improvement needed here, I think you should file a separate issue for that.